packages/blocks/src/page-block/edgeless/components/resize/resize-manager.ts

```diff
@@ -210,6 +254,38 @@ export class HandleResizeManager {
       dp.x += deltaX;
       dp.y += deltaY;
 
+      if (
```

Comment:
Thanks for the contribution! But here I think "resizing on edge" refers more to following effect:

https://github.com/toeverything/blocksuite/assets/7312949/89554cbb-567c-4914-ad60-ba5d3c88cd6f

With draggable edges, we don't have to add more visible handles :)

Reply:
By the way, can I understand that the edge can be dragged in horizontal and vertical directions, but the circular handles are not displayed on the edge?

Reply:
Yep!